atspi-common/src/events/mod.rs

```diff
@@ -42,6 +42,30 @@ use crate::{
 };
 //use atspi_macros::try_from_zbus_message;
 
+pub(crate) fn signatures_are_eq(lhs: &Signature, rhs: &Signature) -> bool {
```

Comment:
Please add some simple tests for this function. It *is* already covered by the existing, more complex tests, but if an issue is found, I'd rather a more specific test fails.